package/MDAnalysis/lib/pkdtree.py

```diff
@@ -230,3 +232,207 @@ def get_indices(self):
           neighbors for the last query points and search radius
         """
         return self._indices
+
+
+class Periodic_cKDTree(object):
```

Comment:
We need to replace the existing pkdtree rather than just add another option here